core/src/main/java/de/danoeh/antennapod/core/sync/SynchronizationSharedPreferenceKeys.java

```diff
@@ -0,0 +1,13 @@
+package de.danoeh.antennapod.core.sync;
+
+public interface SynchronizationSharedPreferenceKeys {
```

Comment:
Now that you have dedicated classes for getting/setting the preferences, the constants don't need to be accessed from outside these classes, don't they? They can be moved to the classes and made private, then.

Reply:
True story 🍸

Reply:
Moving the constants and making them private is still open :)